Makefile

```diff
@@ -148,6 +148,15 @@ endif
 	@$(PHP) ./cli/manipulate.translation.php -a ignore -k $(key) -l $(lang)
 	@echo Key ignored.
 
+.PHONY: i18n-ignore-unmodified-key
+i18n-ignore-unmodified-key: ## Ignore all unmodified translation keys for the selected language
+ifndef lang
+	@echo To ignore a key, you need to provide a language in the "lang" variable.
+	@exit 10
+endif
```

Comment:
Could you use the Make way of dealing with errors? If you don't know how to do it, you could read my PR (#3062)

Reply:
Agree, your PR looks cleaner.  My baseline did not have your PR and wanted to stay consistent.  Will revise